polaris-react/src/components/InContextLearning/InContextLearning.tsx

```diff
@@ -18,21 +17,22 @@ interface Position {
 }
 
 interface Props {
-  children?: React.ReactElement[];
   steps: InContextLearningStep[];
-  onDismiss(): void;
+  children?: React.ReactElement[];
+  onDismiss?(): void;
```

Comment:
There was [some conversation yesterday ](https://shopify-hack-days.slack.com/archives/C03BFM4D54Y/p1652125473077789?thread_ts=1652125260.072959&cid=C03BFM4D54Y)and I don't think we want this to be optional